.eslintrc.json

```diff
@@ -8,6 +8,8 @@
 	"root": true,
 	"rules": {
 		"indent": "off",
-		"operator-linebreak": "off"
+		"operator-linebreak": "off",
+		"no-unused-vars": "off" 
```

Comment:
Why did you add this? We shouldn't have unused variables

Reply:
okay I will remove that